performance/tasks/cumul_smoke.yaml

```diff
@@ -0,0 +1,143 @@
+{% set flavor = flavor or "m1.tiny" %}
+{% set image = image or "^cirros.*uec$" %}
+{% set availability_zone = availability_zone or "nova" %}
+{% set start_cidr = start_cidr or "10.2.0.0/24" %}
+{% set ext_network = ext_network or "public" %}
+---
+
+  KeystoneBasic.create_and_delete_role:
+    -
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+
+  NovaServers.boot_and_delete_server:
```

Comment:
Our goal is to run as few as possible resources in cloud producing as small as possible load. 

- Do not use scenarios that provides such small amount of atomic actions. It would be better to use here something like boot_and_bounce server that can collect all metrics related to all VM operations.